<launch>
  <node name="path_generation" pkg="path_generation" type="path_generation_node" output="screen">
    <param name="active" value="true"/>

    <param name="frequency" value="30"/>
    <param name="init_pose_x" value="0.0"/>
    <param name="init_pose_y" value="0.0"/>
    <param name="init_pose_yaw" value="$(eval 0.0 * 3.1415 / 180.0)"/>
    <param name="resolution" value="100"/>
  
    <param name="path_topic" value="nav_path"/>
    <param name="goal_topic" value="nav_goal"/>
    <param name="pose_topic" value="ekf_pose"/>
  </node>
</launch>
